File: moban_java/src/main/java/org/jdk/project/dto/config/TaskDto.java
package org.jdk.project.dto.config;

import lombok.Builder;
import lombok.Getter;

@Getter
@Builder
public class TaskDto {
  private Long id;
  private String taskName;
  private Long robotId;
  private String robotName;
  private String roomCode;
  private String deptName;
  private Long command;
  private String commandCn;
  private String priority;
  private String executeType;
  private String executeDay;
  private String executeTime;
  private String taskType;
  private String taskSubtype;
  private String taskMode;
  private String directExecution;
  private String isReturn;
  private Integer enable;
  private String remark;
}

File: backend/src/main/kotlin/org/jdk/project/dto/config/TaskDto.kt
package org.jdk.project.dto.config

data class TaskDto(
    var id: Long? = null,
    var taskName: String? = null,
    var robotId: Long? = null,
    var robotName: String? = null,
    var roomCode: String? = null,
    var deptName: String? = null,
    var command: Long? = null,
    var commandCn: String? = null,
    var priority: String? = null,
    var executeType: String? = null,
    var executeDay: String? = null,
    var executeTime: String? = null,
    var taskType: String? = null,
    var taskSubtype: String? = null,
    var taskMode: String? = null,
    var directExecution: String? = null,
    var isReturn: String? = null,
    var enable: Int? = null,
    var remark: String? = null,
)
